test/features/steps/steps.py

```diff
@@ -30,6 +30,14 @@ def go_to_dataset(context, name):
     when_i_visit_url(context, '/dataset/' + name)
 
 
+@step(u'I go to dataset "{name}" comments')
+def go_to_data_request_comments(context, name):
```

Comment:
Misleading function name.

Reply:
@ThrawnCA Updated the method name to match step name